src/components/Missions/Mission.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { toggleMissionJoin } from '../../redux/missions/missions';
```

Comment:
- [ ] There are a couple of errors in the console. Kindly fix them.

Reply:
fixed